content/en/blog/_posts/2023-04-11-kmsv2-beta.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: blog
+title: "Kubernetes 1.27: KMS V2 Moves to Beta"
+date: 2023-04-11
```

Comment:
```suggestion
date: 2023-05-01
```

Reply:
Please also change the path `content/en/blog/_posts/2023-04-11-kmsv2-beta.md` to match the new date.

Reply:
done!